tests/bud.bats

```diff
@@ -262,3 +262,23 @@ load helpers
   [ "$status" -eq 0 ]
   [ "$output" = "" ]
 }
+
+@test "bud-shell" {
+  target=alpine-image
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/shell
+  run buildah --debug=false inspect --type=image --format '{{.Docker.Config.Shell}}' ${target}
+  echo $output
+  [ "$status" -eq 0 ]
+  [ "$output" = "[/bin/sh -c]" ]
+  run buildah --debug=false config --shell "/bin/bash" --shell "\-c" ${target}
```

Comment:
This doesn't seem to jive with the man page, and it looks kind of weird.  Would it be simpler to parse it as a shell command, like we do for the `--cmd` and `--entrypoint` options?